CMakeLists.txt

```diff
@@ -40,6 +40,7 @@ option(VTR_ENABLE_CAPNPROTO "Enable capnproto binary serialization support in VP
 option(WITH_BLIFEXPLORER "Enable build with blifexplorer" OFF)
 option(WITH_LIBRTLNUMBER "Enable build with librtlnumber" OFF)
 
+option(ODIN_BUILD "Enable building odin" ON)
```

Comment:
I would change the name to option(WITH_ODIN"Enable build with odin" ON)

Reply:
Done